### ISSUE TITLE
Call for maintainers for charts with a single OWNER

### ISSUE THREAD

**Issue by maorfr** [helm/charts#13867]
Hello,

As maintaining this repository is growing harder by the day, this is a call for maintainers!

This issue is about 'recruiting' maintainers for charts that only have a single maintainers. The list of charts can be found at the end of this message, along with the existing maintainer of that chart.

If you wish to be added as a maintainer to one of the charts in this issue, please open a PR to add yourself to the `OWNERS` file.
In addition, maintainers in Chart.yaml and OWNERS should match.

We would suggest to include in the PR references to past contributions to the chart you are applying for.

If you are a maintainer and know someone who fits this (and is interested) - feel free to approach them!

Once a PR exists, please add a comment referencing this issue, and ping one of the [maintainers](https://github.com/helm/charts/blob/master/OWNERS#L1) to get added as a read-only user to the helm/charts repository.

Thanks in advance to everyone who applies! We need your help.

All new OWNERS should be familiar with the following documents:
https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md
https://github.com/helm/charts/blob/master/CONTRIBUTING.md
https://helm.sh/docs/chart_best_practices/

Here is the list of charts and maintainers:

incubator/azuremonitor-containers
@vishiy

~~incubator/chartmuseum~~
~~@jdolitsky~~

incubator/druid
@maver1ck

~~incubator/gogs~~
~~@obeyler~~

incubator/kafka
@benjigoldberg

incubator/schema-registry
@benjigoldberg

incubator/hoverfly
@tommysitu

incubator/buzzfeed-sso
@darioblanco

incubator/rundeck
@dwardu89

stable/aerospike
@okgolove 

stable/apm-server
@mumoshu

stable/auditbeat
@cpanato

stable/bookstack
@verwilst

stable/centrifugo
@cpanato

stable/cerebro
@davidkarlsen

~~stable/chaoskube~~
~~@linki~~

stable/chronograf
@jackzampolin

~~stable/cockroachdb~~
~~@a-robinson~~

stable/dex
@desaintmartin

stable/distributed-jmeter
@pedrocesar-ti

~~stable/drone~~
~~@christian-roggia~~

stable/efs-provisioner
@hareku

~~stable/graphite~~
~~@monotek~~

stable/helm-exporter
@sstarcher

~~stable/home-assistant~~
~~@billimek~~

stable/horovod
@cheyang

stable/influxdb
@jackzampolin

stable/jaeger-operator
@cpanato

stable/k8s-spot-rescheduler
@komljen

stable/kube-state-metrics
@tariq1890 

stable/magic-ip-address
@mumoshu

stable/magic-namespace
@krancour

~~stable/nextcloud~~
~~@chrisingenhaag~~

stable/nfs-server-provisioner
@kiall

~~stable/node-problem-detector~~
~~@max-rocket-internet~~

stable/node-red
@billimek

stable/oauth2-proxy
@miouge1

stable/openebs
@kmova

stable/openiban
@verwilst

stable/prisma
@giacomoguiulfo

stable/prometheus-mongodb-exporter
@steven-sheehy

stable/prometheus-snmp-exporter
@miouge1

stable/redis-ha
@ssalaues

stable/satisfy
@anapsix

stable/schema-registry-ui
@cristian04

stable/sematext-docker-agent
@komljen

~~stable/sonarqube~~
~~@rjkernick~~

~~stable/sonatype-nexus~~
~~@rjkernick~~

stable/spark-history-server
@yuchaoran2011

stable/tensorflow-notebook
@cheyang

stable/tensorflow-serving
@cheyang

stable/vault-operator
@mlaccetti

~~stable/velero~~
@hectorj2f

stable/graylog
@KongZ

stable/prometheus-nats-exporter
@okgolove 

stable/clamav
@zakkg3

stable/sentry
@okgolove 

~~stable/goldpinger~~
~~@okgolove~~

**Comment by monotek** [comment]
@maorfr
My stable/graphite chart is already deprecated and moved to:
https://github.com/kiwigrid/helm-charts/tree/master/charts/graphite
We also have 2 maintainers now :-)

**Comment by a-robinson** [comment]
Sorry, this question is probably way out of date since this repo is no longer under the Kubernetes org, but is it ok these days to have an OWNER on a chart that isn't a member of the Kubernetes github organization?

**Comment by yuchaoran2011** [comment]
I'm the sole owner of stable/spark-history-server. I have a pending [PR](https://github.com/helm/charts/pull/9642/) to add my colleague who is familiar with the chart as a co-owner. I need someone to /lgtm my PR for it to be merged.

**Comment by maorfr** [comment]
@yuchaoran2011 would you be able to create a PR with only the addition of @skonto?
then you can pair on all pending PRs...

**Comment by a-robinson** [comment]
Thanks @maorfr! That answers my question but creates the new question of how someone gets added as a collaborator. That can be resolved on #13889, though.

**Comment by caarlos0** [comment]
I can help on stable/prometheus-nats-exporter, have several commits on nats-exporter itself as well although I'm not part of the org.

**Comment by okgolove** [comment]
@caarlos0 hello! Feel free to add you to OWNERS file.

**Comment by caarlos0** [comment]
should the CI have this closed?

**Comment by caarlos0** [comment]
oh, on #14530 the description says `fixes #13867 (comment)`...

**Comment by hectorj2f** [comment]
Oh yeah! That was my PR :/. Sorry

**Comment by darioblanco** [comment]
I am the maintainer of `incubator/buzzfeed-sso`. Lately, we had several MRs from the community, but I can't merge them as I am not a trusted collaborator. I have created an issue to become one here https://github.com/helm/charts/issues/14682 . 

Even if I am quite active with the development and testing of that chart, anybody else who wants to help and take ownership is definitely welcomed :)

**Comment by billimek** [comment]
[stable/minecraft] chart doesn't even have a single owner.  I'm happy to pitch-in on this one.  Is anyone else interested to co-own the chart? @gtaylor? @itzg ?

**Comment by itzg** [comment]
@billimek, since I don’t actively use Helm I’d be happy for you to take point on https://github.com/helm/charts/tree/master/stable/minecraft and would be good with co-owning since I can help keep the chart lockstep updated with the upstream image.

**Comment by darioblanco** [comment]
I have added myself as an OWNER to `stable/hubot` (I should have done it when I created the chart :facepalm:). It is currently stuck until approved by a helm member.

https://github.com/helm/charts/pull/14784

**Comment by sstarcher** [comment]
Fixing a huge bug in dex and adding myself as a maintainer - https://github.com/helm/charts/pull/15050

**Comment by caarlos0** [comment]
stable/consul seems to have only one maintainer as well... cc/  @lachie83

**Comment by yurrriq** [comment]
I’d love someone, especially someone who runs on GCE, to join me on stable/cluster-autoscaler :)

**Comment by r0fls** [comment]
Is it possible to become a repo wide maintainer? I would only use this power for simple changes like https://github.com/helm/charts/pull/15954

I have generic helm knowledge more so than chart specific knowledge. I've made numerous contributions to various helm charts.

**Comment by DandyDeveloper** [comment]
@maorfr I've been helping out on a number of issues/charts, most prominently `stable/redis-ha` as of late. 

I'd like to be a contributor, but I need to be added to the helm approved list of maintainers. 

Do I need to go through that process first, or do I made the suggestion here? 

@ssalaues can probably attest for the recent updates in that chart.

**Comment by maorfr** [comment]
@DandyDeveloper go ahead and submit a PR. once it is approved by the maintainers of the chart, we'll get you added as a trusted collaborator. thanks!

**Comment by Neustradamus** [comment]
Can you look for this ticket?
- https://github.com/helm/charts/issues/10768

**Comment by asherf** [comment]
I volunteer to help w/ https://github.com/helm/charts/tree/master/stable/prometheus-cloudwatch-exporter
I'll submit a PR adding myself to owners.

**Comment by asherf** [comment]
> I volunteer to help w/ https://github.com/helm/charts/tree/master/stable/prometheus-cloudwatch-exporter
> I'll submit a PR adding myself to owners.

@maorfr https://github.com/helm/charts/pull/18093 looks like I need an invite to the helm org.

**Comment by zanhsieh** [comment]
@asherf 
If helm members do not respond within a week, please find them / directly message them on kubernetes slack channel. They usually are there but just busy; you know, typical engineer life.

**Comment by Vlaaaaaaad** [comment]
I'd like to ask the same thing @r0fls asked in https://github.com/helm/charts/issues/13867#issuecomment-516476436:

> Is it possible to become a repo wide maintainer? I would only use this power for simple changes like #15954
I have generic helm knowledge more so than chart specific knowledge. I've made numerous contributions to various helm charts.

I wanted to get some Prometheus changes merged in but PRs have not been merged for while due to very few OWNERS.

I do work extensively with Helm and I believe I can adequately review changes that are not very app-specific. If somebody modifies some intense config for a chart containing an app I never used I am clearly not the person to review. But anything lower than that I think I could do.

**Comment by Neustradamus** [comment]
Any news for https://github.com/helm/charts/issues/10768?

**Comment by maorfr** [comment]
Please do not hijack this thread as well.
